entries/a/also.com.json

```
@@ -0,0 +1,17 @@
+{
+  "ALSO Schweiz AG": {
```

Since the authentication system is the same for all regions that ALSO serves, the country can be skipped.
```suggestion
  "ALSO": {
```